lxd/storage/drivers/driver_lvm.go

```diff
@@ -526,6 +528,61 @@ func (d *lvm) Update(changedConfig map[string]string) error {
 		d.logger.Debug("Thin pool volume renamed", logger.Ctx{"vg_name": d.config["lvm.vg_name"], "thinpool": d.thinpoolName(), "new_thinpool": changedConfig["lvm.thinpool_name"]})
 	}
 
+	size, ok := changedConfig["size"]
+	if ok {
+		sizeBytes, err := d.roundedSizeBytesString(size)
```

Comment:
Was there a specific reason you used this function here? In `Create` its not used, and the user's requested size is pass directly to LVM.

Reply:
There's no need for this. I've changed it to `units.ParseByteSizeString`.